core/src/main/java/com/linecorp/armeria/common/logging/JsonLogFormatter.java

```diff
@@ -131,6 +133,14 @@ public String formatRequest(RequestOnlyLog log) {
             objectNode.put("startTime",
                            TextFormatter.epochMicros(log.requestStartTimeMicros()).toString());
 
+            if (RequestLogProperty.SESSION.isAvailable(flags)) {
+                final ObjectNode connectionNode =
+                        maybeCreateConnectionTimings(log.connectionTimings(), objectMapper);
+                if (connectionNode != null) {
+                    objectNode.set("clientConnection", connectionNode);
```

Comment:
nit:
```suggestion
                    objectNode.set("connection", connectionNode);
```
because the line usually contains `LoggingClient`?